src/_typography.scss

```diff
@@ -87,3 +96,7 @@ $font-variants: (
     text-decoration: none;
   }
 }
+
+div {
```

Comment:
I don't understand why you added this div

Reply:
It is legacy code. Removed in last commit